apps/cli/src/index.ts

```diff
@@ -136,6 +136,22 @@ cli.action(async () => {
             cli.outputHelp();
             continue;
         }
+        
+        // Show help message if the user asks for it
```

Comment:
Should we remove, or update this comment?

Reply:
Updated the comment, thanks !